shared/templates/create_mount_options.py

```diff
@@ -32,7 +32,17 @@ def generate(self, target, path_info):
                         "MOUNTPOINT":       mount_point,
                         "MOUNTOPTION":       re.sub(' ', ',', option),
                     },
-                    "./ansible/mount_options{0}.yml", point_id + '_' + option_id
+                    "./ansible/mount_option{0}.yml", point_id + '_' + option_id
```

Comment:
I don't know which one is correct, I just see that they are inconsistent and we should make sure we use the right one.